### ISSUE TITLE
extractRelationship extractId is called without passing the relationship modelClass

### ISSUE THREAD

**Issue by makepanic** [201-created/ember-data-hal-9000#50]
I don't know if it's expected behavior but we have access to the relationship modelClass so why not pass it to `extractId`?

https://github.com/201-created/ember-data-hal-9000/blob/master/addon/mixin.js#L205

**Comment by Ramblurr** [comment]
Yup, I had to override the mixin function manually to get this. Would be useful.
